seqr/utils/elasticsearch/es_search.py

```diff
@@ -115,6 +115,11 @@ def update_dataset_type(self, dataset_type, keep_previous=False):
             indices.update(new_indices)
             self._indices = list(indices)
         else:
+            if not new_indices:
```

Comment:
this should already be on dev, so it looks like you messed up pulling in dev somehow? Did you like try to cherry pick or something?

Reply:
I think it maybe is a bug of GitHub. It disappears after I refreshed the base branch.